package.json

```diff
@@ -43,103 +47,92 @@
+    "@hapi/good": "9.0.1",
+    "@hapi/hapi": "^20.2.2",
+    "@hapi/inert": "7.0.0",
+    "@hapi/vision": "7.0.0",
+    "@mojaloop/central-services-logger": "^11.0.1",
+    "@mojaloop/ml-testing-toolkit-shared-lib": "^13.0.0",
+    "@mojaloop/sdk-standard-components": "17.1.1",
+    "@now-ims/hapi-now-auth": "2.1.0",
+    "@types/socket.io": "^3.0.1",
+    "adm-zip": "^0.5.9",
     "atob": "2.1.2",
-    "aws-sdk": "^2.1009.0",
-    "axios": "^0.21.4",
-    "blipp": "4.0.1",
-    "chai": "^4.2.0",
-    "cli-table3": "0.6.0",
-    "commander": "4.0.0",
-    "cookie-parser": "^1.4.5",
+    "axios": "^0.27.2",
+    "chai": "^4.3.6",
+    "cookie-parser": "^1.4.6",
     "cookies": "0.8.0",
     "cors": "^2.8.5",
-    "docdash": "1.1.1",
-    "dotenv": "^8.6.0",
-    "express": "^4.17.1",
-    "express-validator": "^6.13.0",
+    "dotenv": "^16.0.1",
+    "express": "^4.18.1",
+    "express-validator": "^6.14.2",
     "fs": "0.0.1-security",
     "handlebars": "^4.7.7",
-    "hapi-auth-bearer-token": "^6.1.4",
+    "hapi-auth-bearer-token": "^8.0.0",
     "hapi-openapi": "^3.0.0",
-    "hapi-swagger": "^14.2.5",
-    "ilp-packet": "^2.2.0",
-    "js-yaml": "^3.13.1",
-    "json-refs": "^3.0.13",
-    "json-rules-engine": "5.0.3",
-    "lodash": "^4.17.19",
-    "mongoose": "^6.0.11",
+    "hapi-swagger": "^14.5.5",
+    "ilp-packet": "^3.1.3",
+    "js-yaml": "^4.1.0",
+    "json-refs": "^3.0.15",
+    "json-rules-engine": "6.1.2",
+    "lodash": "^4.17.21",
+    "mongoose": "^6.4.6",
     "multer": "^1.4.3",
-    "mustache": "3.1.0",
+    "mustache": "4.2.0",
     "mv": "^2.1.1",
     "node-dir": "0.1.17",
     "node-strings": "^1.0.2",
     "npm-run-all": "4.1.5",
-    "openapi-backend": "4.2.0",
-    "parse-strings-in-object": "1.2.0",
-    "passport": "0.4.1",
+    "openapi-backend": "5.3.0",
+    "parse-strings-in-object": "1.6.0",
+    "passport": "0.6.0",
     "passport-jwt": "4.0.0",
     "path": "^0.12.7",
-    "postman-collection": "^4.1.0",
-    "postman-sandbox": "^4.0.5",
+    "postman-collection": "^4.1.4",
+    "postman-sandbox": "^4.0.8",
     "rc": "1.2.8",
     "request": "^2.88.2",
     "request-promise-native": "1.0.8",
     "request-to-curl": "^0.1.6",
     "selectn": "^1.1.2",
-    "socket.io": "^2.3.0",
-    "socket.io-client": "^2.3.0",
-    "uuid": "8.1.0",
-    "uuid4": "1.1.4",
-    "vm": "0.1.0",
-    "ws": "^7.5.5"
+    "socket.io": "^4.5.1",
+    "socket.io-client": "^4.5.1",
+    "uuid": "8.3.2",
+    "uuid4": "2.0.2",
+    "ws": "^8.8.1"
   },
   "devDependencies": {
-    "@types/jest": "24.0.22",
-    "eslint": "^8.0.1",
-    "eslint-plugin-import": "^2.25.2",
-    "get-port": "5.0.0",
-    "jest": "^27.3.0",
-    "jest-junit": "^13.0.0",
-    "jsdoc": "^3.6.10",
-    "nodemon": "^2.0.13",
-    "npm-audit-resolver": "^2.3.1",
-    "npm-check-updates": "11.3.0",
-    "nyc": "^15.0.0",
-    "parse-strings-in-object": "1.2.0",
+    "@types/jest": "^28.1.6",
+    "audit-ci": "^6.3.0",
+    "eslint": "^8.20.0",
```

Comment:
Any particular reason for ESLint here since we are using standard?

Reply:
Removed, thanks